.github/workflows/test01-new-new.yml

```diff
@@ -51,37 +51,30 @@ jobs:
       
       - name: Test the ERC20 value transfer
         run: cd tests/value-transfer/ERC20 && bash run_testcase.sh
-        
-      - name: Logging the kend.out of EN node when job was failed
-        if: failure()
-        run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T EN-0 cat /klaytn/log/kend.out
-
-      - name: Logging the kscnd.out of SCN node when job was failed
-        if: failure()
-        run: |
-          cd servicechain-docker
-          docker-compose -f docker-compose-servicechain.yml --env-file $DOCKER_COMPOSE_ENV_FILE exec -T SCN-0 cat /klaytn/log/kscnd.out
-
-      - name: Logging the kcnd.out of CN node when job was failed
+      
+      - name: Copy log files in the failed-logs directory in case of failure
         if: failure()
```

Comment:
Unconditionally archiving logs, regardless of success and failure, seems reliable unless it makes much cost. What do you think?